### DISCUSSION TITLE
"Merging via command line" instructions very large

### DISCUSSION THREAD

**Discussion by SkylerWilliams** [community/community#15936] [Pull Requests] [answered]
It seems that some instructions were added to the bottom of the Pull Request for merging via the command line. While this may be useful for some at times, I believe that it will more often be not what the user is looking for or trying to do. As such, the size and obtrusiveness of these instructions feels a little disproportionate to their usefulness. Could the detailed copy-pasteable information be hidden under a dropdown or something of the sort, so it is still easily available for those who need it but not taking up non-trivial space for those who do not? Thanks!

![Very Large Instructions](https://user-images.githubusercontent.com/8980010/166336868-b6f99ae1-80da-4b7a-ad93-4a868b1b8297.png)

**Comment by andrewrondeau-optirtc** [comment]
I agree. IMO, just turn it off. It feels like putting in a button for just one person at the inconvenience of everyone else.

This seems like something that should be part of an FAQ or documentation. (With an inconspicuous link on the pull request page itself.)

**Comment by mskelton** [comment]
I'm also not a fan of the new steps, people who want to merge via the command line likely already know or they can easily find out with Google/StackOverflow.  Please provide a way to turn off this message.

**Comment by SkylerWilliams** [comment, marked as answer]
This appears to have been updated to be a small link next to the Merge button.
